src/dymaptic.GeoBlazor.Core/Components/Symbols/TextSymbol.cs

```diff
@@ -71,13 +112,93 @@ public TextSymbol()
+    public bool? Kerning { get; set; }
+    
+    /// <summary>
+    ///     The height of the space between each line of text. Only applies to multiline text.
+    ///     This property can be considered as a multiplier of the default value of 1.0 (e.g. a value of 2.0 will be two times the height of the default height). The range of possible values is: 0.1 - 4.0. If a value of 0 is specified, the default value of 1.0 will be used.
+    ///     Default Value: 1.0
+    /// </summary>
+    [Parameter]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public double? LineHeight { get; set; }
+    
+    /// <summary>
+    ///     The maximum length in points for each line of text. This value is a string expressing size in points or pixels (e.g. "12px", "12pt"), which defaults to points.
+    ///     The default value is 192 points. The range of possible values is: 32px - 512px.
+    ///     If text extends farther than the lineWidth value, then the line will break at the whitespace before the text that extends past the limit if possible, and a new line will be created.
+    /// </summary>
+    /// <remarks>
+    ///     Known Limitations:
+    ///     - This property is currently not supported in 3D SceneViews.
+    ///     - The default value is subject to change in future releases.
+    /// </remarks>
+    [Parameter]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public string? LineWidth { get; set; }
+    
+    /// <summary>
+    ///     Determines whether every character in the text string is rotated. Default value is false.
+    /// </summary>
+    /// <remarks>
+    ///     This property is currently not supported in 3D SceneViews.
+    /// </remarks>
+    [Parameter]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public bool? Rotated { get; set; }
 
     /// <summary>
     ///     The text string to display in the view.
     /// </summary>
     [Parameter]
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public string? Text { get; set; }
+    
+    /// <summary>
+    ///     Adjusts the vertical alignment of the text. Default value is Baseline.
+    /// </summary>
+    /// <remarks>
+    ///     This property only applies when TextSymbol is not used for labeling purposes. The verticalAlignment for labels is inferred from the labelPlacement value.
+    /// </remarks>
+    [Parameter]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public VerticalAlignment? VerticalAlignment { get; set; }
+    
+    /// <summary>
+    ///     The offset on the x-axis in points. This value is a string expressing size in points or pixels (e.g. "12px", "12pt"), which defaults to points.
+    /// </summary>
+    /// <remarks>
+    /// This property is currently not supported in 3D SceneViews.
+    /// </remarks>
+    [Parameter]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public string? XOffset { get; set; }
```

Comment:
In looking at the `TextSymbol` docs, I noticed that a number of these new properties are autocast, are not supported in 3d Sceneviews or are only applicable when `TextSymbol` is not used for labeling, is this why they are not added in the main `TextSymbol` constructor?

Reply:
No, they aren't added to the constructor because I forgot to update the constructor 🙂